react/src/index.js

```diff
@@ -80,44 +81,15 @@ Sentry.init({
       // Additional SDK configuration goes in here, for example:
       colorScheme: 'system',
     }),
-    new Sentry.metrics.MetricsAggregator(),
-    new Sentry.BrowserProfilingIntegration(),
-    new Sentry.BrowserTracing({
-      tracingOrigins: tracingOrigins,
-      tracePropagationTargets: tracingOrigins,
-      routingInstrumentation: Sentry.reactRouterV6Instrumentation(
-        useEffect,
-        useLocation,
-        useNavigationType,
-        createRoutesFromChildren,
-        matchRoutes
-      ),
-      beforeNavigate: (context) => {
```

Comment:
@andreiborza to confirm, is the reason for removing the beforeNavigate block that it's outdated and no longer necessary?

Reply:
yes, the newer `reactRouterV6BrowserTracingIntegration` doesn't have an option for that and from what I can tell, it's (just?) doing route parametrization which the newer integration does out of the box.

Could you also chime in here please @mydea?

Reply:
sorry, saw this too late! This should not be needed here, if needed you can use `beforeStartSpan` instead, see https://docs.sentry.io/platforms/javascript/tracing/instrumentation/automatic-instrumentation/#beforestartspan